src/NServiceBus.WindowsPerformanceCounters/Performance/ReceiveStatisticsFeature.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.WindowsPerformanceCounters
+{
+    using System.Threading.Tasks;
+    using Features;
+
+    class ReceiveStatisticsFeature : Feature
+    {
+        protected override void Setup(FeatureConfigurationContext context)
+        {
+            var logicalAddress = context.Settings.LogicalAddress();
+            var performanceDiagnosticsBehavior = new ReceivePerformanceDiagnosticsBehavior(logicalAddress.EndpointInstance.Endpoint);
+
+            context.Pipeline.Remove("ReceivePerformanceDiagnosticsBehavior");
```

Comment:
This can be removed since we should disable the https://github.com/Particular/NServiceBus/blob/c9d979e7fef88e8a6a9ea2f39ba253eed756b957/src/NServiceBus.Core/Performance/ReceiveStatisticsPerformanceCounters.cs#L12 feature instead